Source/Scene/Cesium3DTileset.js

```diff
@@ -1158,10 +1170,16 @@ define([
          */
         clippingPlaneOffsetMatrix : {
             get : function() {
-                if (this._useBoundingSphereForClipping) {
-                    return this._clippingPlaneOffsetMatrix;
+                if (!defined(this._clippingPlaneTransformMatrix)) {
+                    return Matrix4.IDENTITY;
                 }
-                return this.root.computedTransform;
+
+                if (this._recomputeClippingPlaneMatrix) {
+                    Matrix4.multiply(this.root.computedTransform, this._clippingPlaneOffsetMatrix, this._clippingPlaneTransformMatrix);
+                    this._recomputeClippingPlaneMatrix = false;
+                }
+
+                return this._clippingPlaneTransformMatrix;
```

Comment:
The naming is a little confusing. What is the difference between `clippingPlaneOffsetMatrix` and `clippingPlaneTransformMatrix`, and why is `clippingPlaneTransformMatrix` returned for a getter called `clippingPlaneOffsetMatrix`. See if there's a more descriptive name for what is `_clippingPlaneOffsetMatrix`.